cumulusci.yml

```diff
@@ -0,0 +1,5 @@
+project:
+    name: Weathervane
+    package:
+        name: Weathervane
+        namespace: indicatorbar
```

Comment:
include an explicit api version here - since the valueset name change in the .object is breaking-y